ZenPacks/zenoss/ZenPackLib/lib/libexec/ZPLCommand.py

```diff
@@ -227,10 +227,25 @@ def lint(cls, filename):
         except Exception, e:
             DEFAULTLOG.exception(e)
 
+    def validate_zenpack_name(self, zenpack_name):
+        """Ensure that ZenPack name conforms with convention"""
+        zenpack_name_parts = zenpack_name.split('.')
```

Comment:
If we are working with ZP name parts below, we can use something similar to the following construction:

```
try:
    zp_name_1, zp_name_2, zp_name_3 = zenpack_name.split('.')
except ValueError:
    sys.exit('ZenPack name must conform ... ')
```